### ISSUE TITLE
On open : System.TypeInitializationException -> Microsoft.ApplicationInsights.Extensibility.TelemetryConfiguration

### ISSUE THREAD

**Issue by benizz0** [carfup/XTBPlugins.PersonalViewsMigration#51]
Hello,

I have an unhandled error when opening the tool :
Probably an issue with the telemetry tool.

Error:

```
Consultez la fin de ce message pour plus de détails sur l'appel du débogage
juste-à-temps (JIT) à la place de cette boîte de dialogue.

************** Texte de l'exception **************
System.TypeInitializationException: Une exception a été levée par l'initialiseur de type pour 'Microsoft.ApplicationInsights.Extensibility.TelemetryConfiguration'. ---> System.MissingMethodException: Méthode introuvable : 'Boolean System.Diagnostics.Activity.get_ForceDefaultIdFormat()'.
   à Microsoft.ApplicationInsights.Extensibility.TelemetryConfiguration.<>c.<.cctor>b__13_0()
   à Microsoft.ApplicationInsights.ActivityExtensions.TryRun(Action action)
   à Microsoft.ApplicationInsights.Extensibility.TelemetryConfiguration..cctor()
   --- Fin de la trace de la pile d'exception interne ---
   à Microsoft.ApplicationInsights.Extensibility.TelemetryConfiguration.get_Active()
   à Carfup.XTBPlugins.AppCode.LogUsageManager..ctor(PersonalViewsMigration pvm)
   à Carfup.XTBPlugins.PersonalViewsMigration.PersonalViewsMigration.PersonalViewsMigration_Load(Object sender, EventArgs e)
   à System.Windows.Forms.UserControl.OnLoad(EventArgs e)
   à System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   à System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   à System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   à System.Windows.Forms.Control.CreateControl()
   à System.Windows.Forms.Control.WmShowWindow(Message& m)
   à System.Windows.Forms.Control.WndProc(Message& m)
   à System.Windows.Forms.Form.WmShowWindow(Message& m)
   à System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

```

The tool still manages to open, but numerous unhandled errors reappear during use.
Example:

```
************** Texte de l'exception **************
System.NullReferenceException: La référence d'objet n'est pas définie à une instance d'un objet.
   à Carfup.XTBPlugins.PersonalViewsMigration.PersonalViewsMigration.<LoadUsersIntoListView>b__23_1(RunWorkerCompletedEventArgs e)
   à XrmToolBox.Extensibility.Worker.<>c__DisplayClass4_0.<WorkAsync>b__0(Object s, RunWorkerCompletedEventArgs e)
   à System.ComponentModel.BackgroundWorker.OnRunWorkerCompleted(RunWorkerCompletedEventArgs e)
```

Certain features are no longer available. 
I am unable to disable the statistics.

```
System.NullReferenceException: La référence d'objet n'est pas définie à une instance d'un objet.
   à Carfup.XTBPlugins.PersonalViewsMigration.PersonalViewsMigration.SaveSettings(Boolean closeApp)
   à Carfup.XTBPlugins.Forms.Options.buttonOk_Click(Object sender, EventArgs e)
   à System.Windows.Forms.Control.OnClick(EventArgs e)
   à System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   à System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   à System.Windows.Forms.Control.WndProc(Message& m)
   à System.Windows.Forms.ButtonBase.WndProc(Message& m)
   à System.Windows.Forms.Button.WndProc(Message& m)
   à System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```
version PersonalViewsMigration : 1.2023.5.13
version XrmToolBox : 1.2023.12.68
CRM connection type : onpremise (v8.2.28.11)
système : windows 11 (Insider preview 23H2, build : 22635.2915) 

Similar issue : [https://github.com/microsoft/ApplicationInsights-dotnet/issues/1223](https://github.com/microsoft/ApplicationInsights-dotnet/issues/1223)

**Comment by carfup** [comment]
Hi,

Thanks for the input, I discovered this issue few days ago, I'll have to update all my plugins.
Hoping that I can do that within the next few days.

I'll keep you updated.

**Comment by benizz0** [comment]
Hi,

Fixed by 1.2024.1.14, thank you !
